test/app/presenters/track_test.rb

```diff
@@ -7,20 +7,34 @@ class PresentersTrackTest < Minitest::Test
   def setup
     @track = ExercismWeb::Presenters::Track.new("test_track")
     @trackler_track = Object.new
+
     @track.stubs(:trackler_track).returns(@trackler_track)
     @track.stubs(:fallback_topic_content).returns("Default topic content")
```

Comment:
Good points. I reworked the tests in https://github.com/exercism/exercism.io/pull/3475/commits/827a210d380561a2f2f6913f50081cea27a5fce5.